examples/db-templates/mongodb-persistent-template.json

```diff
@@ -169,6 +177,11 @@
   ],
   "parameters": [
     {
+      "name": "DEPLOYMENT_LIMIT_SIZE",
+      "description": "Maximum amount of memory allowed by OpenShift Quota System",
```

Comment:
Not a very good description.  Also, the name needs some work.  Probably better to call this MEMORY_LIMIT and the description needs to be very clear what the impact to the user is.  I wouldn't talk about OpenShift Quota System - that has no meaning to end users.